libs/net/dal/Services/UserService.cs

```diff
@@ -128,7 +128,10 @@ public IPaged<User> Find(UserFilter filter)
     public User? FindByUsername(string username)
     {
         return this.Context.Users
-            .Where(u => u.Username.ToLower() == username.ToLower()).FirstOrDefault();
+            .Include(u => u.MediaTypes)
```

Comment:
Include the blocked sources and media types so that they will be added to every search.